transmart-core-api/src/main/groovy/org/transmartproject/core/config/CountsThresholdResource.groovy

```diff
@@ -0,0 +1,14 @@
+package org.transmartproject.core.config
+
+
+interface CountsThresholdResource {
```

Comment:
This is not needed. Instead of injecting `CountsThresholdResource` to call for the function, you can also just use `@Value` instead:
```groovy
    @Value("${patientCountThreshold}")
    long patientCountThreshold = 0
```